neo/rawio/neuralynxrawio.py

```diff
@@ -179,10 +180,14 @@ def _parse_header(self):
                     # an event channel
                     # each ('event_id',  'ttl_input') give a new event channel
                     self.nev_filenames[chan_id] = filename
-                    data = np.memmap(
-                        filename, dtype=nev_dtype, mode='r', offset=HEADER_SIZE)
-                    internal_ids = np.unique(
-                        data[['event_id', 'ttl_input']]).tolist()
+
+                    if (os.path.getsize(filename) <= HEADER_SIZE):
+                        self._empty_nev.append(filename)
+                        data = np.zeros((0,), dtype=dtype)
```

Comment:
I don't the dtype variable.
I would do:
`dtype=dtype` > `dtype=nev_dtype`

Reply:
Yes, you are right. Fixed it in the latest commit.